tests/integration_test.rs

```diff
@@ -12,12 +12,14 @@ fn platform() -> Result<(), String> {
     println!("release = {}", uname.release());
     println!("version = {}", uname.version());
     println!("machine = {}", uname.machine());
+    println!("osname = {}", uname.osname());
```

Comment:
maybe we could add per platform tests as in this case, we know what we will get, wdyt?

Reply:
I'll look at it tonight.

Reply:
I removed the redundant WinOS test, and added platform-specific tests of `osname()` for 'unix' and 'windows'.